net.studio08.xbplay.yml

```diff
@@ -16,7 +16,8 @@ finish-args:
   - --share=network
   - --device=all
   - --filesystem=/run/udev:ro
-  - --filesystem=/home/deck/.local/share/Steam/userdata
+  - --filesystem=/home/deck/.local/share/Steam
+  - --filesystem=/home/deck/.steam
```

Comment:
`/home/deck`. How does it work when there is not such directory?

Reply:
Yeah, I don't understand what's the point of this. Flathub works with all distros, not just Steam Deck.

Reply:
Whoops, sorry for all the commits, hopefully, that wasn't pinging you guys 😟

Flathub works with all distros, not just Steam Deck.

One of the features of this app is that it adds custom shortcuts and artwork to games installed via Steam. In order to do this, I need access to the local steam installation directory. If not on a Steam Deck, then the user will not get the option to add to Steam, so this feature is essentially disabled. If on the steam deck, then I need this permission otherwise I cannot add custom shortcuts.

How does it work when there is no such directory

Nothing breaks, if that directory doesn't exist then I disable the feature that uses this directory (adding Steam shortcuts).

Reply:
@barthalion / @hfiguiere 
I could give file access to the entire host system, but that seems like overkill. Quick question on this actually guys, can I give access to the user home folder? So for example, /home/<username>/ABC

On the steam deck, the steam path will always be `/home/deck/.steam
On other forms of linux, it will be `/home/<user>/.steam`

If I want to support adding shortcuts for non steam deck linux devices, is my only option to require permission to the entire `/home` folder? I tried to give it permission to `~/.steam` but that didn't seem to work.

Reply:
Oh I may be mistaken, `~/.steam` may be working actually, that is better than using `home/deck/.steam`.